### PR TITLE
Cash 794

### PR DESCRIPTION
**Pull request by mcstover**

### CODE REVIEW

File: src/pages/Lend/LendByCategoryPage.vue [hunk 1]
```diff
@@ -137,7 +150,7 @@ export default {
 			categorySetting: [],
 			categorySetId: '',
 			itemsInBasket: [],
-			imageEnhancementExperimentVersion: '',
+			imageEnhancementExperimentVersion: null,
```
Comment (mcstover) [inline src/pages/Lend/LendByCategoryPage.vue]: I'll finish cleaning this up (including commented code below) later in the sprint.

File: src/pages/Lend/LendByCategoryPage.vue [hunk 2]
```diff
@@ -152,6 +165,8 @@ export default {
 			showExpandableLoanCards: false,
 			rightArrowPosition: undefined,
 			leftArrowPosition: undefined,
+			hasFavoriteCountry: false,
+			favoriteCountryExpVersion: 'control',
```
Comment (ksekhri) [inline src/pages/Lend/LendByCategoryPage.vue]: Not necessary to change but we could default this to `null` to increase readability

**Comment by mcstover** [comment]
FYI @BoulderBrains here's the final state.

File: src/components/LoansByCategory/FavoriteCountryLoans.vue [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+		};
+	},
+	apollo: {
+		preFetch(config, client) {
+			return client.query({
+				query: favoriteCountryQuery
+			}).then(data => {
+				const favoriteCountry = _get(data, 'data.my.recommendations.topCountry');
+				// skip if no favorite country
+				if (favoriteCountry === undefined) return null;
```
Comment (BoulderBrains) [inline src/components/LoansByCategory/FavoriteCountryLoans.vue]: 👍

File: src/pages/Lend/LendByCategoryPage.vue [hunk 4]
```diff
@@ -200,6 +221,21 @@ export default {
...
+			if (this.showFavoriteCountryRow) {
+				const favoriteCountryLoans = _get(this.$refs, 'favoriteCountries.favoriteCountryLoans');
+				if (favoriteCountryLoans !== undefined && favoriteCountryLoans.length > 0) {
+					_each(favoriteCountryLoans, (loan, loanIndex) => {
+						loanIds.push({
+							r: -1,
+							p: loanIndex + 1,
+							c: 99,
+							l: loan.id
+						});
```
Comment (BoulderBrains) [inline src/pages/Lend/LendByCategoryPage.vue]: Looking forward to talking to you about how this rpcl is working. Looks pretty interesting.

File: src/pages/Lend/LendByCategoryPage.vue [hunk 5]
```diff
@@ -200,6 +221,21 @@ export default {
...
+					});
```
Comment (BoulderBrains) [inline src/pages/Lend/LendByCategoryPage.vue]: r:c:p:l:, this is nice. 👍

Comment (mcstover) [reply src/pages/Lend/LendByCategoryPage.vue]: This pushes data into our custom page state event alongside all the other rows. It is then compared with an interaction event containing the same information when an add to basket button is clicked. Thus we can build a map of which rows and positions are most affective.

**Pull request merged**